src/styles/header.css

```diff
@@ -0,0 +1,47 @@
+header {
+  display: flex;
+  align-items: center;
+  background-color: #fff;
+  padding: 1.7rem 6.25vw;
+}
+
+h1 {
+  font-size: 1.875rem;
+  font-weight: bold;
+  color: #0290ff;
```

Comment:
- [ ] I would suggest to make your Logo as clickable `cursor:pointer` and when you click if it directs you to home page it also would be good option for it.

Reply:
Done! 😀